srcpkgs/radicale/files/README.voidlinux

```diff
@@ -0,0 +1,6 @@
+## Create a new htpasswd file with the user "user1"
+$ htpasswd -c /etc/radicale/users user1
+$ chmod 640 /etc/radicale/users
+
+## Add another user
+$ htpasswd /etc/radicale/users user2
```

Comment:
I think this is documented upstream and not void specific, is it?

Reply:
Removed.